en/computer_vision/collision_prevention.md

```diff
@@ -2,7 +2,7 @@
 
 *Collision Prevention* may be used to automatically slow and stop a vehicle before it can crash into an obstacle.
 
-It can be enabled for multicopter vehicles in [Position mode](../flight_modes/position_mc.md), and at time of writing requires a companion computer.
+It can be enabled for multicopter vehicles in [Position mode](../flight_modes/position_mc.md), and can use sensor data from an offboard companion computer, a rangefinder attached to the flight controller, or both (fused).
```

Comment:
@mrivi HOw does "fusing" work? ie this could just be taking lots of messages from different orientations and passing them into avoidance code, or it could be that this does some magic if multiple sensors cover the same orientation ... or both.

Reply:
Awesome! Added a note about that: https://github.com/PX4/px4_user_guide/pull/537/commits/e9e9bee3c171b122f13d1bee9e75c49e6c875f1e